README.md

```
@@ -16,7 +16,7 @@ This vulnerability is further aggravated by mismanagement in disaster response.
 Effective disaster management during critical situations can significantly reduce the impact of any disasters. We propose a centralized system designed to coordinate and manage disaster response in an organized and efficient manner.  
 Our platform will facilitate coordination among various volunteer organizations, streamline relief distribution, organize fund collection, and manage other essential activities needed to manage a disaster.
 
-## Features
+## Prominent Features
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)